ALVRClient/FFR.swift

```diff
@@ -85,9 +103,9 @@ struct FFR {
         let eyeHeightRatioAligned = optimizedEyeHeight / Float(optimizedEyeHeightAligned)
         
         return FoveationVars(
-            enabled: data.enable_foveation,
-            targetEyeWidth: data.view_width,
-            targetEyeHeight: data.view_height,
+            enabled: true,
```

Comment:
Why are you forcing FFR on?

Reply:
We're protected by the `guard` at the top of this function. The previous code behaved this way too.

Reply:
Ah cool, i missed that.